libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -540,6 +540,13 @@ bool AP_BattMonitor::capacity_remaining_pct(uint8_t &percentage, uint8_t instanc
     return false;
 }
 
+/// capacity_remaining_pct_mavlink - returns battery percentage if percentage is valid, -1 otherwise
```

Comment:
Sorry, but let's keep the mavlink stuff in the mavlink library here.

that might be a `GCS_MAVLINK::battery_remaining_pct` for example.

Reply:
Sure. I added this method in mavlink library. Please review these changes and let me know if something needs to be changed :)